src/components/Tooltip/TooltipContent.tsx

```diff
@@ -0,0 +1,42 @@
+/* tslint:disable:no-unused-variable */
+import * as React from 'react';
+/* tslint:enable:no-unused-variable */
+import { BaseComponent } from '../../common/BaseComponent';
+import { ITooltipProps } from './Tooltip.Props';
+import { ITooltipState } from './Tooltip';
+import { getId } from '../../Utilities';
+import './Tooltip.scss';
+
+export class TooltipContent extends BaseComponent<ITooltipProps, ITooltipState> {
```

Comment:
I don't think there is anything in this class that merits it being its own thing; can't you just have this content in the Tooltip component? The only reason to split it out is because you're doing something special in lifetime methods, or using refs.

And even then, we may want to have a Callout wrapper so that you don't have to invent 2 classes for 1 component just to work around the quirk.